x-pack/test/functional/fixtures/kbn_archiver/timelion.json

```diff
@@ -0,0 +1,52 @@
+    "timeFieldName": "@timestamp",
+    "title": "logstash-*"
+  },
+  "coreMigrationVersion": "8.0.0",
```

Comment:
This was exported from master/8.0. We should export from 7.x so it's able to be backported.

Reply:
Have tried changing the `coreMigrationVersion": "7.14.0" ` Hope it works.